packages/common/core-interfaces/src/events/listeners.ts

```diff
@@ -48,7 +47,6 @@ export type Listeners<T extends object> = {
  * @sealed @public
  */
 export interface Listenable<TListeners extends object> {
-	/* eslint-enable @fluid-internal/fluid/no-hyphen-after-jsdoc-tag -- false positive
 	/**
 	 * Register an event listener.
```

Comment:
Note for reviewers: Somehow this comment seems to have caused API-Extractor to consider `on` to be undocumented. Removing this causes a _correct_ update to the API reports.